app/models/course.rb

```diff
@@ -18,7 +18,7 @@ def postrequisites
   end
 
   def schedulable?
-    course_instances.any?(&:schedulable?)
+    course_instances.any?{|course_instance| course_instance.try(:schedulable?)}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.